apps/web-giddh/src/app/voucher/voucher.component.ts

```diff
@@ -8478,4 +8449,145 @@ export class VoucherComponent implements OnInit, OnDestroy, AfterViewInit, OnCha
             this.isCreatingNewAccount = false;
         }
     }
+
+    /**
+     * Loads the stock variants
+     *
+     * @private
+     * @param {string} stockUniqueName Uniquename of the stock
+     * @memberof VoucherComponent
+     */
+    private loadStockVariants(stockUniqueName: string, index?: number): void {
+        this._ledgerService.loadStockVariants(stockUniqueName).pipe(
```

Comment:
add takeUntil

Reply:
This method will be called multiple times therefore takeUntil is not used. HTTP obseravble will complete as soon as it emits the value.